content/blog/2022/08/camel-karavan-3.18.1/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Karavan Preview Release 3.18"
+date: 2022-08-19
+authors: [mgubaidullin]
+categories: ["Releases","Tooling"]
+preview: "Karavan 3.18 - cloud-native integration toolkit"
+---
+
+![logo](toolkit.png)
+
+The next significant milestone is Karavan development. Karavan still comes with two main options: VS Code extenson and cloud-native integration toolkit for Kubernetes/OpenShift (FKA cloud/serverless mode). 
+
+Disclamer:
+
+Karavan version is aligned with Camel version.
```

Comment:
The version of Karavn is now aligned to the Camel release that the UI designer is based upon. This makes it easier to understand which Camel version Karavan can be used as design editor.